source/manuals/accessibility.html.md.erb

```diff
@@ -11,13 +11,13 @@ Government services must be accessible to everyone. This includes anyone with a
 
 Building a service with accessibility in mind not only allows those with access needs to use your service, it also improves the service for everyone else. An accessibility problem with a website can be something that affects everyone, not just people who can only access the web with a keyboard or screen reader.
 
-Government services are legally required to be accessible. This means services must comply with the international WCAG 2.1 AA accessibility standard. This requirement has applied to all new and existing public sector websites since 2020, and to native applications since June 2021.
+Government services are legally required to be accessible. This means services must comply with the international WCAG 2.2 AA accessibility standard. This requirement applies to all new and existing public sector websites and native applications.
```

Comment:
```suggestion
Government services are legally required to be accessible. This means services must comply with the international WCAG 2.2 AA accessibility standard. This requirement applies to all new and existing public sector websites and mobile applications.
```

Reply:
One minor suggestion to reflect the wording in the [accessibility regulations](https://www.legislation.gov.uk/uksi/2018/952).